In [10]:
import pandas as pd
import plotly.express as px

df = pd.read_csv("https://gist.githubusercontent.com/jbrobst/56c13bbbf9d97d187fea01ca62ea5112/raw/e388c4cae20aa53cb5090210a42ebb9b765c0a36/active_cases_2020-07-17_0800.csv")

fig = px.choropleth(
    df,
    geojson="https://gist.githubusercontent.com/jbrobst/56c13bbbf9d97d187fea01ca62ea5112/raw/e388c4cae20aa53cb5090210a42ebb9b765c0a36/india_states.geojson",
    featureidkey='properties.ST_NM',
    locations='state',
    color='active cases',
    color_continuous_scale='Reds'
)

fig.update_geos(fitbounds="locations", visible=False)

fig.show()

In [ ]:
import pandas as pd
import plotly.express as px

df = pd.read_csv("https://gist.githubusercontent.com/jbrobst/56c13bbbf9d97d187fea01ca62ea5112/raw/e388c4cae20aa53cb5090210a42ebb9b765c0a36/active_cases_2020-07-17_0800.csv")

fig = px.choropleth(
    df,
    geojson="https://gist.githubusercontent.com/jbrobst/56c13bbbf9d97d187fea01ca62ea5112/raw/e388c4cae20aa53cb5090210a42ebb9b765c0a36/india_states.geojson",
    featureidkey='properties.ST_NM',
    locations='state',
    color='active cases',
    color_continuous_scale='Reds'
)

fig.update_geos(fitbounds="locations", visible=False)

fig.show()

In [15]:
import pandas as pd
import plotly.express as px

thaiPop = pd.read_excel('Thai_population_collected.xlsx')
thaiPop.head()

Province   CODE  Year  Population
0  Bangkok Metropolis  TH-10  2553   5701394.0
1  Bangkok Metropolis  TH-10  2554   5674843.0
2  Bangkok Metropolis  TH-10  2555   5673560.0
3  Bangkok Metropolis  TH-10  2556   5686252.0
4  Bangkok Metropolis  TH-10  2557   5692284.0

In [3]:
thaiPop.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 770 entries, 0 to 769
Data columns (total 4 columns):
 #   Column      Non-Null Count  Dtype  
---  ------      --------------  -----  
 0   Province    770 non-null    object 
 1   CODE        770 non-null    object 
 2   Year        770 non-null    int64  
 3   Population  769 non-null    float64
dtypes: float64(1), int64(1), object(2)
memory usage: 24.2+ KB


import plotly.express as px
import plotly.graph_objects as go

fig = px.choropleth(thaiPop, locations="CODE",
                    color="Population", # lifeExp is a column of gapminder
                    hover_name="Province", # column to add to hover information
                    animation_frame="Year", # column on which to animate
                    color_continuous_scale=px.colors.cyclical.IceFire #or .sequential.Plasma
                   )
                    
fig.update_layout(
    # add a title text for the plot
    title_text = 'Thai Population 2553 - 2562',
    geo_scope = 'asia'
    )


fig.show()

In [16]:
fig = px.choropleth(thaiPop,
                    geojson="https://raw.githubusercontent.com/cvibhagool/thailand-map/master/thailand-provinces.geojson",
                    featureidkey='properties.NAME_1',
                    locations='Province',
                    color='Population',
                    hover_name="Province", # column to add to hover information
                    animation_frame="Year", # column on which to animate
                    color_continuous_scale=px.colors.cyclical.IceFire # or .sequential.Plasma
                    
)

fig.update_geos(fitbounds="locations", visible=False)

fig.show()